tasks/connectors/aqua/lib/aqua_helper.rb

```diff
@@ -5,30 +5,77 @@
 module Kenna
   module Toolkit
     module AquaHelper
+      SAAS_AUTH_URL = "https://api.cloudsploit.com/v2/signin"
+      WP_URL_API    = "https://prov.cloud.aquasec.com/v1/envs"
+
       def aqua_get_token(aqua_url, username, password)
-        print_debug "Getting Auth Token"
-        aqua_auth_api = "#{aqua_url}/api/v1/login"
-        # auth_headers = { "content-type" => "application/json",
-        #            "accept" => "application/json" }
-        # auth_body = { "id" => "administrator",
-        #              "password" => "My@rvgicmx1" }
-
-        @headers = { "Content-Type" => "application/json" }
-        payload = {
-          "id": username.to_s,
```

Comment:
I don't have access to the API docs to know: is `email` on line 31 an acceptable substitute for `id` when using Aqua on-prem?

Reply:
Me too, so I've reverted id when using Aqua on-prem